packages/@tinacms/react-toolbar/src/components/Toolbar.tsx

```diff
@@ -22,10 +22,11 @@ import { Plugin } from '@tinacms/core'
 import { Form } from '@tinacms/forms'
 import { FieldMeta } from '@tinacms/fields'
 import { Button, TinaResetStyles } from '@tinacms/styles'
+import { ScreenPlugin, ScreenPluginModal } from '@tinacms/react-screens'
```

Comment:
so glad we split screens out from the sidebar 🙌

Reply:
Heckin right.